### ISSUE TITLE
Accessibility Testing Feedback

### ISSUE THREAD

**Issue by emahadzimustafic** [askaalyy/web-design-2-independent-project#6]
**Usability**
- You can add highlighting the current page you’re on in the nav (https://learn-the-web.algonquindesign.ca/topics/navigation/#highlighting-the-current-page)
- You have focus styles! Awesome!
- Skip links are included! Just need to make them invisible + have them show when using the keyboard
- Aria label needed on the “learn more” and “click here” button on main to describe what the user is clicking to
- Screen reader doesn’t read your roles even though you have them, not sure how to fix this

**Images&Media**
- Shorten your alt text to a few words (eg. a white present icon in a tan circle). I also noticed that the word graphic is spelled graphi in your alt-text, so make sure to go over spelling.
- The “hi there” text on page 1 has a spelling error (says expirence instead of experience)
- On the work page, one image isn’t working (under Joaquin phoenix spread - love the spread btw)
- Images are extremely scaled and distort/shrink horizontally, so make sure to fix that (i think width 100% or width auto might work?)
- ADD A FAVICON :) once you make a logo for yourself

**Content** 
- Info is organized and flows well
- I noticed that when I click on a certain page from the nav, that nav option is at the top of the page and not in the nav anymore. I’m not sure if this is intentional 
![nav on work page](https://user-images.githubusercontent.com/77508090/145520567-2af6aa2b-a818-47b4-8083-42231d843b68.JPG)
- I only see h1 in the header, so if you do add more text and hierarchy, add h2 and h3 as well!
